tests/expectations.json

```
@@ -31183,6 +31149,7 @@
   "language/white-space/string-nbsp.js": "FAIL",
   "language/white-space/string-space.js": "FAIL",
   "language/white-space/string-vertical-tab.js": "FAIL",
+  "language\\expressions\\object\\11.1.5-0-1.js": "FAIL",
```

```suggestion
  "language/expressions/object/11.1.5-0-1.js": "FAIL",
```

Ugh, Windows file paths. I'll have to fix the test262 runner to not do this.